dev/sdk_php.go

```diff
@@ -99,8 +116,10 @@ func (t PHPSDK) Publish(
 // Bump the PHP SDK's Engine dependency
 func (t PHPSDK) Bump(ctx context.Context, version string) (*dagger.Directory, error) {
 	version = strings.TrimPrefix(version, "v")
-	content := fmt.Sprintf(`<?php /* Code generated by dagger. DO NOT EDIT. */ return '%s';
-`, version)
+	content := fmt.Sprintf(
+		"<?php\n\n/* Code generated by dagger. DO NOT EDIT. */return '%s';",
```

Comment:
You removed a space there. And should also probably add a newline, according to the linter:
```
FILE: /src/sdk/php/src/Connection/version.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 3 | ERROR | [x] Expected 1 newline at end of file; 0 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```
```suggestion
		"<?php\n\n/* Code generated by dagger. DO NOT EDIT. */\nreturn '%s';\n",
```